doc/src/manual/profile.md

```diff
@@ -211,44 +225,31 @@ Let's discuss these arguments in order:
     @profile Profile.print(STDOUT, data) # Prints the previous results
     Profile.print()                      # Prints results from Profile.print()
     ```
-  * The first keyword argument, `format`, was introduced above. The possible choices are `:tree` and
-    `:flat`.
-  * `C`, if set to `true`, allows you to see even the calls to C code. Try running the introductory
-    example with `Profile.print(C = true)`. This can be extremely helpful in deciding whether it's
-    Julia code or C code that is causing a bottleneck; setting `C=true` also improves the interpretability
-    of the nesting, at the cost of longer profile dumps.
-  * Some lines of code contain multiple operations; for example, `s += A[i]` contains both an array
-    reference (`A[i]`) and a sum operation. These correspond to different lines in the generated
-    machine code, and hence there may be two or more different addresses captured during backtraces
-    on this line. `combine=true` lumps them together, and is probably what you typically want, but
-    you can generate an output separately for each unique instruction pointer with `combine=false`.
-  * `cols` allows you to control the number of columns that you are willing to use for display. When
-    the text would be wider than the display, you might see output like this:
 
-    ```
-    33 inference.jl; abstract_call; line: 645
-      33 inference.jl; abstract_call; line: 645
-        33 ...rence.jl; abstract_call_gf; line: 567
-           33 ...nce.jl; typeinf; line: 1201
-         +1 5  ...nce.jl; ...t_interpret; line: 900
-         +3 5 ...ence.jl; abstract_eval; line: 758
-         +4 5 ...ence.jl; ...ct_eval_call; line: 733
-         +6 5 ...ence.jl; abstract_call; line: 645
-    ```
+The keyword arguments can be any combination of:
 
-    File/function names are sometimes truncated (with `...`), and indentation is truncated with a
-    `+n` at the beginning, where `n` is the number of extra spaces that would have been inserted,
-    had there been room. If you want a complete profile of deeply-nested code, often a good idea is
-    to save to a file and use a very wide `cols` setting:
+  * `format` which was introduced above. The possible choices are `:tree` and
+    `:flat`.
+  * If `C` is `true`, backtraces from C and Fortran code are shown (normally they are excluded).
+  * If `combine` is `true` (default), instruction pointers are merged that correspond to the same line of code.
+  * `maxdepth` can be used to limit the depth of printing in `:tree` format
+  * `sortedby` can be used to control the order in `:flat` format. `:filefuncline` (default) sorts by the source
```

Comment:
Likewise, perhaps "`sortedby` controls the ...", and perhaps "`mincount` limits the printout..." below?